README.rst

```diff
@@ -97,6 +97,7 @@ For Developers
 
 2. Build and Test::
 
+    $ cd Parsl # navigate to the root directoty of the project
```

Comment:
there is a spelling mistake in directory

on most unix system, the directory would be `parsl` with a lowercase p, not uppercase P.

Reply:
Resolved. @benclifford thanks for the quick feedback.